index.js

```diff
@@ -100,40 +114,71 @@ const serializers = {
   },
   ReleaseEvent: (item) => {
     return `🚀 ${capitalize(item.payload.action)} release ${toUrlFormat(
-      item.payload.release.name
-        ? item.payload.release.name
-        : item.payload.release.tag_name
+      item
     )} in ${toUrlFormat(item.repo.name)}`;
   },
 };
 
 Toolkit.run(
   async (tools) => {
-    // Get the user's public events
     tools.log.debug(`Getting activity for ${GH_USERNAME}`);
-    const events = await tools.github.activity.listPublicEventsForUser({
-      username: GH_USERNAME,
-      per_page: 100,
-    });
+
+    let page = 1;
+    let events = [];
+
+    while (events.length < MAX_LINES) {
+      let data = [];
+
+      try {
+        // Get the user's public events
+        const resp = await tools.github.activity.listPublicEventsForUser({
+          username: GH_USERNAME,
+          per_page: 100,
+          page: page,
```

Comment:
```suggestion
          page,
```

Reply:
i'm personally not a fan of the shortened version in this instance as all the other parameters are explicitly named or set with a fixed value. it looks kind of strange to me when there is a line missing the key value pair. but if you have a strong preference for it i can change it.

Reply:
I find the object shorthand syntax handy and always prefer it. I see the point here not to mix it up and be explicit. If you ask me, I'd always go for the syntax sugar. If we can use the ES6 semantics, then why not leverage it? Feel free to take the call here.